pkg/neutronapi/dbsync.go

```diff
@@ -73,15 +75,13 @@ func DbSyncJob(cr *neutronv1beta1.NeutronAPI, scheme *runtime.Scheme) *batchv1.J
 			},
 		},
 	}
-	initContainerDetails := common.InitContainer{
+	initContainerDetails := InitContainer{
 		ContainerImage: cr.Spec.ContainerImage,
-		DatabaseHost:   cr.Spec.DatabaseHostname,
+		DatabaseHost:   cr.Spec.DatabaseInstance,
```

Comment:
You will want to use `cr.Status.DatabaseHostname` as that should be have been populated with the proper hostname during your `reconcileInit`: https://github.com/openstack-k8s-operators/neutron-operator/blob/379c6f50180863180b745d6f4672701f35c85dc3/controllers/neutronapi_controller.go#L241

Reply:
Sure will check and fix